rocAL/rocAL/source/decoders/image/fused_crop_decoder.cpp

```diff
@@ -123,9 +107,7 @@ Decoder::Status FusedCropTJDecoder::decode(unsigned char *input_buffer, size_t i
             x1 = std::uniform_int_distribution<int>(0, original_image_width - crop_width)(_rand_gen);
             y1 = std::uniform_int_distribution<int>(0, original_image_height - crop_height)(_rand_gen);
         }
-        else
-        {
-
+        else {
```

Comment:
Check the google coding guidelines, applicable for all the else conditions

Reply:
Changed